src/components/select/select.component.html

```diff
@@ -5,6 +5,7 @@
     [tags]="_value$ | async"
     (tagsChange)="_multipleOptionSelected($event)"
     [(input)]="input"
+    [attr.aria-required]="required"

```

Comment:
I don't think this does anything because the component wrapper doesn't receive focus. Did you mean to use the `required` input that you added?